File: yurt-system/yurt-system-server/src/main/java/com/github/liulus/yurt/system/service/ParamService.java
package com.github.liulus.yurt.system.service;

import com.github.liulus.yurt.system.model.dto.SysParamQo;
import com.github.liulus.yurt.system.model.entity.SysParam;
import com.github.liulus.yurt.convention.data.Page;

/**
 * User : liulu
 * Date : 17-9-17 下午2:53
 * version $Id: ParamService.java, v 0.1 Exp $
 */
public interface ParamService {

    Page<SysParam> findPageList(SysParamQo qo);

    SysParam findById(Long id);

    SysParam findByCode(String code);

    Long insert(SysParam param);

    void update(SysParam param);

    void delete(Long... ids);
}

File: yurt-system/yurt-system-server/src/main/kotlin/com/github/liulus/yurt/system/service/ParamService.kt
package com.github.liulus.yurt.system.service

import com.github.liulus.yurt.convention.data.Page
import com.github.liulus.yurt.system.model.dto.SysParamQo
import com.github.liulus.yurt.system.model.entity.SysParam

/**
 * User : liulu
 * Date : 17-9-17 下午2:53
 * version $Id: ParamService.java, v 0.1 Exp $
 */
interface ParamService {
    fun findPageList(qo: SysParamQo?): Page<SysParam?>?
    fun findById(id: Long?): SysParam?
    fun findByCode(code: String?): SysParam?
    fun insert(param: SysParam?): Long?
    fun update(param: SysParam?)
//    fun delete(vararg ids: Long?)
}